daemon/state_test.go

```diff
@@ -187,6 +188,12 @@ func (ds *DaemonSuite) generateEPs(baseDir string, epsWanted []*e.Endpoint, epsM
 }
 
 func (ds *DaemonSuite) TestReadEPsFromDirNames(c *C) {
+	oldDatapath := ds.d.datapath
```

Comment:
Maybe add a comment that because this test reads EPs from directories, it needs the real linux datapath.